task1/task2.js

```diff
@@ -0,0 +1,27 @@
+const fs = require("fs");
+const csv = require("csvtojson");
+
+const csvFilePath = "./csv/nodejs-hw1-ex1.csv";
+const outputFilePath = "./output.txt";
+const readable = fs.createReadStream(csvFilePath);
+const writable = fs.createWriteStream(outputFilePath);
+const onError = (error) => console.log(error);
+
+readable
```

Comment:
You can use stream.pipeline for simplicity: https://nodejs.org/api/stream.html
E.g.:
```pipeline(
  fs.createReadStream('archive.tar'),
  zlib.createGzip(),
  fs.createWriteStream('archive.tar.gz'),
  (err) => {
    if (err) {
      console.error('Pipeline failed.', err);
    } else {
      console.log('Pipeline succeeded.');
    }
  }
);```

Reply:
done